ethereum/test/MessageQueue.t.sol

```diff
@@ -363,7 +363,8 @@ contract MessageQueueTest is TestHelper {
         );
     }
 
-    function test_submit_transaction() public {
+    // Test skipped, to enable it remove the skip_ prefix
```

Comment:
Please also add `TODO` to find this comment later

Reply:
Fixed